src/Misc/layoutroot/config.sh

```diff
@@ -8,7 +8,7 @@ fi
 source ./env.sh
 
 if [[ "$1" == "remove" ]]; then
```

Comment:
I'm also curious here why "remove" is supported. Do you have context on this? Even if we need to support it for OSX/Linux, I'm still skeptical it is required for Windows.